pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -9,7 +9,7 @@
   cmake, util-linux, linkFarm, symlinkJoin, which, pybind11, removeReferencesTo,
 
   # Build inputs
-  numactl,
+  darwin, numactl,
```

Comment:
```suggestion
  numactl,
  CoreServices, libobjc,
```

Reply:
I originally tried this but got this error:
```
error: Function called without required argument "CoreServices" at /Users/malo/Code/nixpkgs/pkgs/development/python-modules/pytorch/default.nix:13
```

Reply:
Ah, I think I can fix that by explicitly providing it as argument to `callPackage` in `pkgs/top-level/python-packages.nix`. Other packages do the same thing.

Reply:
Done.

Reply:
Thank you. This is done to simplify overrides.